rustfmt.toml

```diff
@@ -0,0 +1,5 @@
+max_width = 100
+hex_literal_case = "Lower"
+imports_granularity = "Crate"
+group_imports = "StdExternalCrate"
+reorder_imports = true
```

Comment:
The one thing I'll say about this is that I tried it for the book, and it generally worked but it did also mangle all of my inline code snippers too; things went from eg

```rust
let  foo = || {
    2
};
fn main() {
    something
}
```

to

```rust
let  foo = || {
2
};
fn main() {
something
}
```

That may have been wrap comments, or it may have been the rule that converts `/**` to `///` comments; worth checking :)